x/consortium/client/cli/tx.go

```diff
@@ -0,0 +1,211 @@
+package cli
```

Comment:
I didn't find unittest of this cli codes. 
How about adding unittest of thease cli commands?

You can reference how to developing the cli unittest in https://github.com/line/lbm-sdk/blob/main/x/bank/client/cli/cli_test.go

Reply:
I will make a new issue on this.
If you agree, please approve this PR :)